code/renderers/svelte/template/cli/js/Button.stories.svelte

```diff
@@ -0,0 +1,31 @@
+<script context="module" lang="ts">
```

Comment:
This shouldn't be `lang="ts"` should it?

Reply:
Good catch. I started by copying the ts version. Fixed